pkg/asset/installconfig/azure/validation_test.go

```diff
@@ -39,7 +45,9 @@ var (
 	invalidateVirtualNetwork     = func(ic *types.InstallConfig) { ic.Azure.VirtualNetwork = "invalid-virtual-network" }
 	invalidateComputeSubnet      = func(ic *types.InstallConfig) { ic.Azure.ComputeSubnet = "invalid-compute-subnet" }
 	invalidateControlPlaneSubnet = func(ic *types.InstallConfig) { ic.Azure.ControlPlaneSubnet = "invalid-controlplane-subnet" }
-	invalidateRegion             = func(ic *types.InstallConfig) { ic.Azure.Region = "eastus" }
+	invalidateRegion             = func(ic *types.InstallConfig) { ic.Azure.Region = "neverland" }
+	invalidateRegionCapabilities = func(ic *types.InstallConfig) { ic.Azure.Region = "australiacentral2" }
+	invalidateRegionLetterCase   = func(ic *types.InstallConfig) { ic.Azure.Region = "CentralUS" }
```

Comment:
```
az account list-locations -otable | grep australiacentral2
Australia Central 2   -35.3075    149.1244     australiacentral2
```
I think the user might give us a display name, which has spaces, so maybe we should test that too.